src/Illuminate/Validation/composer.json

```diff
@@ -15,7 +15,8 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-filter": "*",
```

Comment:
See usages of `filter_var`, `FILTER_FLAG_EMAIL_UNICODE`, `FILTER_FLAG_IPV4`, `FILTER_FLAG_IPV6`, `FILTER_VALIDATE_EMAIL`, `FILTER_VALIDATE_INT`, `FILTER_VALIDATE_IP`, and `FILTER_VALIDATE_MAC`.